### DISCUSSION TITLE
Neumorphism/Soft UI CSS theme

### DISCUSSION THREAD

**Discussion by InsightfulFuture** [beercss/beercss#158] [Ideas]
I really like apples design aesthetic but can not find a good looking apple based css lib. If [Puppertino](https://codedgar.github.io/Puppertino/examples/index.html) looked better, I would consider using it but it is still very limited. I think the modals would be good to reference for styles though. It would be awesome to have a apple option (as a theme - like `<body class='apple-light|apple-dark'>`. It would be awesome to have more advanced themes like this and beercss doesn't exactly say material design in the name. 🍺

**Comment by InsightfulFuture** [comment]
[KonstaUI](https://konstaui.com/) has apple styled _components_, but that is not a CSS framework. But, the styles look really nice - you could probably base a theme off of it. The modals are also better than puppertino.

**Comment by InsightfulFuture** [comment]
A visionOs theme would be better - [Figma Design FIle](https://www.figma.com/community/file/1253443272911187215/Apple-Design-Resources---visionOS)

**Comment by leonardorafael** [comment]
My knowledge about apple design system is nearly null 😅
